### PR TITLE
Update Readme for 0.4

### PR DESCRIPTION
**Pull request by seongahjo**

### CODE REVIEW

**Review by acktsap — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -18,92 +19,77 @@ You can write countless tests including edge cases by only one instance of the F
...
+```
+
 ## Example
 
 ```java
 
-@Data   // lombok getter, setter
+@Value
 public class Order {
-    @NotNull
```
Comment (mhyeon-lee) [inline README.md:27]: @seongahjo 아까 회의에 못 들어갔었는데요
저는 `javax validation annotation` 에 따라 값이 만들어지는걸 표현하는게 더 좋을거 같은데 어떻게 논의가 되었을까요?

Comment (seongahjo) [reply README.md:27]: @mhyeon-lee 
`fixture-monkey-javax-validation` 의존성 추가 설명을 추가하면 위 `Install`의 의존성 추가 안내와 겹쳐서 제거했습니다.

Comment (seongahjo) [reply README.md:27]: ex.
## Install

### Gradle

```groovy
testImplementation("com.navercorp.fixturemonkey:fixture-monkey-starter:0.4.2")
```

### Maven

```xml
<dependency>
    <groupId>com.navercorp.fixturemonkey</groupId>
    <artifactId>fixture-monkey-starter</artifactId>
    <version>0.4.2</version>
    <scope>test</scope>
</dependency>
```

## Example
### Install
```groovy
testImplementation("com.navercorp.fixturemonkey:fixture-monkey-starter:0.4.2")
testImplementation("com.navercorp.fixturemonkey:fixture-monkey-javax-validaiton:0.4.2")

```

### Maven

```xml
<dependency>
    <groupId>com.navercorp.fixturemonkey</groupId>
    <artifactId>fixture-monkey-starter</artifactId>
    <version>0.4.2</version>
    <scope>test</scope>
</dependency>
<dependency>
    <groupId>com.navercorp.fixturemonkey</groupId>
    <artifactId>fixture-monkey-javax-validation</artifactId>
    <version>0.4.2</version>
    <scope>test</scope>
</dependency>
```

**Review by mhyeon-lee — APPROVED** [review]
